docs/architecture.adoc

```diff
@@ -0,0 +1,162 @@
+= Cluster Logging Architecture
+Overview of cluster logging architecture and designs in progress.
+:imagesdir: architecture
+:toc:
+// Workaround for adminition icons on github.
+ifdef::env-github[]
+:tip-caption: :bulb:
+:note-caption: :information_source:
+:important-caption: :heavy_exclamation_mark:en
+:caution-caption: :fire:
+:warning-caption: :warning:
+endif::[]
+
+:enhancment_proposal:link:https://github.com/openshift/enhancements/blob/master/guidelines/README.md[Enhancement Proposal]
+:enhancment_repo https://github.com/openshift/enhancements/tree/master/enhancements/cluster-logging[Enhancement Repository]
+
+== Introduction
+
+What this is:
+
+* A brief, but _accurate_, overview of the _current_ cluster logging architecture. The link:.[source for this site] is part of the source code repository.
+* A brief overview of design work in progress.
+* A link to more detailed information - see <<Resources>>
+
+What this is not:
+
+* A replacement or duplicate of existing <<resources>> such as JIRA, Bugzilla or the {enhancement_repo}.
+* A replacement for the {enhancment_proposal} process.
+
+This site will hold early notes on upcoming enhancements, when they are too speculative or incomplete to be framed as an {enhancment_proposal}.
+As soon as design goals are clear enough, they move to the {enhancment_repo} and may be linked, but not duplicated, here.
+
+== Architecture Overview
+
+[.thumb]
+.Key to diagrams
+image::legend.svg[]
+
+=== Operators and APIs
+
+[.thumb]
+.Operators and APIs
+image::overview.svg[]
+
+=== Collection and forwarding
+
+[.thumb]
+.Collection and forwarding
```

Comment:
Add a section to show what type of logs define which input app/infra/audit.

Reply:
Done, also elaborated on the overview pictures if you want to check again.